typings/index.d.ts

```diff
@@ -373,7 +373,7 @@ declare module 'klasa' {
 	export abstract class Argument extends AliasPiece {
 		public constructor(client: KlasaClient, store: ArgumentStore, file: string[], directory: string, options?: ArgumentOptions);
 		public aliases: string[];
-		public abstract run(arg: string, possible: Possible, message: KlasaMessage): any;
+		public abstract run(arg: string | undefined, possible: Possible, message: KlasaMessage): any;
```

Comment:
typing prs do not belong in docs prs

Reply:
I didn't add them they must have come from when the PR was rebased to alpha branch. Should i remake the PR?

Reply:
This is just a rebase artifact.

The line already exists as given in https://github.com/dirigeants/klasa/blob/v1.0.0-alpha/typings/index.d.ts#L390

Reply:
Should i remake the PR or just leave it as is?

Reply:
Should remake, yeah.